desktop/src/com/mygdx/game/client/hud/GameRoomHUD.java

```diff
@@ -108,9 +138,28 @@ private void preparePlayerTable() {
     float height = stage.getHeight();
     this.playerTable = uiElementsCreator.createTable((float) (width * 0.1), (float) (height * 0.1));
     uiElementsCreator.setActorWidthAndHeight(this.playerTable, (int) (width * 0.4), (int) (height * 0.8));
-    for (String player : players) {
-      var label = uiElementsCreator.createLabel("Nickname: " + player, 0, 0);
-      uiElementsCreator.addCellToTable(label, playerTable);
+    for (PlayerLobby player : players) {
+      var singlePlayerTable = uiElementsCreator.createTable((float) 0, (float) 0);
+      var label = uiElementsCreator.createLabel("Nickname: " + player.getUserName(), 0, 0);
+      uiElementsCreator.addToTableRow(label, singlePlayerTable);
+
+      var selectBox = uiElementsCreator.createSelectBox();
+      var allCivs = gameConfigAssets.getGameConfigs().getAll(CivilizationConfig.class);
+      selectBox.setItems(allCivs);
+      selectBox.setSelected(gameConfigAssets.getGameConfigs().get(CivilizationConfig.class, player.getCivId()));
+      selectBox.addListener(new ChangeListener() {
+        @Override
+        public void changed (ChangeEvent event, Actor actor) {
+          log.info(selectBox.getSelected().toString());
+          playerInfo.setCivilization(((CivilizationConfig) selectBox.getSelected()).getId());
+          connectService.reconnect();
```

Comment:
może zamiast reconnect to coś innego typu "playerConfigurationChanged" albo coś, bo nie wiem czy to co się dzieje można nazwać reconnectem